Dockerfile_test

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6
+
+COPY requirements.txt .
+COPY test_requirements.txt .
+RUN pip install -r requirements.txt -r test_requirements.txt
+
+WORKDIR /usr/src
+ENV PYTHONPATH "${PYTHONPATH}:/usr/src"
```

Comment:
I guess the purpose for this Docker container is to isolate test dependencies and have a clean enviroment for tests.

1. This can also be achieved in a simpler way (and with a sufficient isolation IMO) using Tox.
2. To make a proper test it would be better to install the SDK instead of linkin the source. Tox does this.

So, to me this looks like unnecessary Dockerization.